docs/howto/index.md

```diff
@@ -25,7 +83,40 @@ Manage opened ports <manage-opened-ports>
 Manage the charm version <manage-the-charm-version>
 Manage the workload version <manage-the-workload-version>
 Manage metrics <manage-metrics>
+```
+
+## Tracing
+
+Ops enables you to trace your charm code and send data to the [Canonical Observability Stack](https://documentation.ubuntu.com/observability/).
+
+```{toctree}
+:maxdepth: 1
+
 Trace your charm <trace-your-charm>
+```
+
+## Charmhub listing
+
+Once your charm is ready for wide production use, your next goal should be to get it publicly listed on Charmhub, so that it is visible in searches.
+
+```{toctree}
+:maxdepth: 1
+
 Make your charm discoverable <make-your-charm-discoverable>
-Legacy how-to guides <legacy/index>
 ```
+
+## Legacy guides
+
+```{toctree}
+:hidden:
+
+legacy/index
+```
+
+Harness is a deprecated framework for writing unit tests. You should migrate to state-transition tests.
+
+- {doc}`legacy/migrate-unit-tests-from-harness`
+
+Hooks-based charms use script files instead of Python code with Ops. This isn't a recommended way of writing charms, so you should migrate to Ops.
```

Comment:
I feel like you could cut the "this isn't a recommended way of writing charms" and just tell them to use ops, like the one above.